src/cli/bin.ts

```diff
@@ -182,6 +241,12 @@ This will output a file named \`report.json\` containing the error messages.`);
       process.exit(0);
     },
   )
+  .check(({ fromVersion, toVersion }) => {
+    if (gte(fromVersion, toVersion)) {
```

Comment:
Actually @antoinetissier this doesn't work, `semver` throws an `Invalid Version` error. I did some testing with it and it seems that when passing `X.Y` as the version, we're missing the patch version for the format to be correct.

It raises the question of what format we accept for the versions. I think we should not be too flexible and only accept `X.Y` formats. Then we can manually append an arbitrary `0` patch version to the versions before `semver` compares them.

What do you think?

Reply:
You can use `semver.coerce`

Reply:
Thanks!